sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -86,7 +86,7 @@ Response<AnalyzeConversationTaskResult> response = client.AnalyzeConversation(
     conversationsProject);
 
 CustomConversationalTaskResult customConversationalTaskResult = response.Value as CustomConversationalTaskResult;
```

Comment:
Line no.20
change from: An existing Text Analytics resource
to: An existing Azure Language Service Resource

Reply:
and also remove:
Note: the new unified Cognitive Language Services are not currently available for deployment.